backend/src/junior/dto/register.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsNotEmpty, IsPhoneNumber } from 'class-validator';
+
+export class RegisterJuniorDto {
+
+    @IsPhoneNumber(null)
```

Comment:
Really nitpicky, but what does the `(null)` do?

Reply:
The method provides the following description:

> Checks if the string is a valid phone number. "region" accepts 2 characters uppercase country code (e.g. DE, US, CH).If users must enter the intl. prefix (e.g. +41), then you may pass "ZZ" or null as region. See google-libphonenumber, metadata.js:countryCodeToRegionCodeMap on github

I provided null as I initially assumed it would be better to require any number to be entered (e.g. if somebody is visiting the country for a prolonged period, but their phone is registered in another country). However, I am now considering simply providing "FI" might be a more suitable option.